src/modals/QRCodeModal/QRCodeModal.jsx

```diff
@@ -74,12 +76,18 @@ const QRCodeModal = ({ onClose, profile }) => {
     setTimeout(() => setCopied(false), 3000)
   }
 
+  const handleClose = () => {
+    const sw = sp.getSw()
+    sw.close()
```

Comment:
you can directly call this.sp.sw.close()

Reply:
this.sp is not accessible here, we are outside of startSync method